backend/src/app/serializers/timeRecord.ts

```diff
@@ -0,0 +1,76 @@
+import * as moment from "moment";
+import {Milestone} from "../models/milestone";
+import {database} from "../../lib/database";
+import {TableNames} from "../../lib/enums";
+import {Request, Response} from "express";
+
+export const createProjects = async (timeRecords: Array<any>) => {
+  let projectId: Milestone;
+  let projects: Array<object> = [];
+  for (let i = 0; i < timeRecords.length / 7; i++) {
+    if (i % 7 == 0) {
+      projectId = await database(TableNames.milestones).where({
```

Comment:
a serializer-ben nem használunk már db-t! Ezt kint, a controller-ben kell elvégezni. Például lekéred egy tömbben és itt csak find-dal keresel benne. De írtam is egyébként:
"szerintem túlgondolod: onnantól, hogy **lekérdtél minden adatot** és csak össze kell rendezni megfelelő formátumba, átadhatod egy serializer függvénynek, ami ezt megoldja. Akár több argumentumot is, ha ez szükséges."